smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/http-uri-conflict-validator.json

```diff
@@ -315,6 +333,120 @@
                     }
                 }
             }
+        },
+        "ns.foo#EndpointPrefixDeconflicts1": {
+            "type": "operation",
+            "traits": {
+                "smithy.api#http": {
+                    "uri": "/endpoint-prefix",
+                    "method": "PUT"
+                },
+                "smithy.api#endpoint": {
+                    "hostPrefix": "prefix."
+                },
+                "smithy.api#idempotent": {}
+            }
+        },
+        "ns.foo#EndpointPrefixDeconflicts2": {
+            "type": "operation",
+            "traits": {
+                "smithy.api#http": {
+                    "uri": "/endpoint-prefix",
+                    "method": "PUT"
+                },
+                "smithy.api#idempotent": {}
+            }
+        },
+        "ns.foo#PatternDeconflicts1": {
+            "type": "operation",
+            "traits": {
+                "smithy.api#http": {
+                    "uri": "/endpoint-prefix-pattern-deconflicts",
+                    "method": "PUT"
+                },
+                "smithy.api#endpoint": {
+                    "hostPrefix": "label."
+                },
+                "smithy.api#idempotent": {}
+            }
+        },
+        "ns.foo#PatternDeconflicts2": {
+            "type": "operation",
+            "traits": {
+                "smithy.api#http": {
+                    "uri": "/endpoint-prefix-pattern-deconflicts",
+                    "method": "PUT"
+                },
+                "smithy.api#endpoint": {
+                    "hostPrefix": "{HostLabel}."
+                },
+                "smithy.api#idempotent": {}
+            },
+            "input": {
+                "target": "ns.foo#HostLabelInputWithPattern"
+            }
+        },
+        "ns.foo#HostLabelInputWithPattern": {
+            "type": "structure",
+            "members": {
+                "HostLabel": {
+                    "target": "smithy.api#String",
+                    "traits": {
+                        "smithy.api#hostLabel": {},
+                        "smithy.api#pattern": "foo",
+                        "smithy.api#required": {}
+                    }
+                }
+            }
+        },
+
```

Comment:
?

Reply:
whoops haha i put in that extra space to make it easier for me to compartmentalize my change but forgot to remove it